tests/components/Addon.test.js

```diff
@@ -17,11 +17,11 @@ const app = {
   "requireOpenmrsVersion": "2.0.0",
   "awareOfModules": [],
   "requiredModules": ["org.openmrs.module.providermanagement",
-    "org.openmrs.module.referencemetadata",
-    "org.openmrs.module.metadatasharing",
-    "org.openmrs.module.emrapi",
-    "org.openmrs.module.metadatadeploy",
-    "org.openmrs.module.idgen"],
+  "org.openmrs.module.referencemetadata",
+  "org.openmrs.module.metadatasharing",
+  "org.openmrs.module.emrapi",
+  "org.openmrs.module.metadatadeploy",
+  "org.openmrs.module.idgen"],
```

Comment:
Good job but you should try to make changes that only affect the ticket. This can also be seen in other part of your code

Reply:
Thanks for the great feedback. I was changing indentation to 2 spaces but also noticed it made the PR look verbose. I shall take that into consideration on future requests

Reply:
In this regard, consider reverting to only changes that directly affect this ticket, then open another ticket to  do  the formatting changes.